application/helpers/previewer_helper.php

```diff
@@ -255,3 +255,9 @@ function format_playtime($seconds)
 	$hours     = ($remainder - $minutes) / 60;
 	return sprintf('%02d:%02d:%02d', $hours, $minutes, $seconds);
 }
+
```

Comment:
Can we split these two into a separate commit? I could be missing something (I guess we can keep the same PR?), but it seems unrelated to the first file.